conformance/lang/expressions/xml-attribute-access-expr/xml_attribute_access_expr.balt

```diff
@@ -0,0 +1,626 @@
+Test-Case: output
```

Comment:
Shall we have separate files for `xml-required-attribute-access-expr` and `xml-optional-attribute-access-expr` tests?

Reply:
Changed